samples/nrf9160/nrf_cloud_rest_device_message/src/main.c

```diff
@@ -344,6 +350,35 @@ static int setup(void)
 	return 0;
 }
 
+int send_hello_world_msg(void)
+{
+	int err;
+	int64_t time_now;
+	char buf[SAMPLE_MSG_BUF_SIZE];
+
+	/* Get the current timestamp */
+	err = date_time_now(&time_now);
+	if (err) {
+		LOG_ERR("Failed to get timestamp for Hello World message");
+		return err;
+	}
+
+	/* Send off a hello world message! */
+	/* This does not match a predefined schema. But that's not a problem! */
+	err = snprintk(buf, SAMPLE_MSG_BUF_SIZE, SAMPLE_MSG, time_now);
+	if (err) {
```

Comment:
snprintf return value:
```
The number of characters that would have been written if n had been sufficiently large, not counting the terminating null character.
If an encoding error occurs, a negative number is returned.
Notice that only when this returned value is non-negative and less than n, the string has been completely written.
```

Reply:
Oops for got this comment, will edit and recommit